src/data/events/visits/pixelmatters.md

```diff
@@ -1,6 +1,6 @@
 ---
```

Comment:
Should these visits markdowns still be here?

Reply:
They are deleted in the program PR